model/Image.php

```diff
@@ -711,7 +711,7 @@ public function cacheFilename($format) {
 		array_shift($args);
 		$folder = $this->ParentID ? $this->Parent()->Filename : ASSETS_DIR . "/";
 
-		$format = $format . base64_encode(json_encode($args, JSON_NUMERIC_CHECK));
+		$format = $format . base64_encode(json_encode(array_map('strval', $args)));
```

Comment:
Good spot, from reading the PR/issue it looks like that was added because `base64_encode()` isn’t url-safe. Looks like whoever gets merged last will need to rebase, but that shouldn’t be an issue :)